cdc/model/changefeed.go

```diff
@@ -66,13 +81,13 @@ func (detail *ChangeFeedDetail) ShouldIgnoreTable(db, tbl string) bool {
 // CDC only supports filtering by database/table now.
 func (detail *ChangeFeedDetail) FilterTxn(t *Txn) {
 	if t.IsDDL() {
-		if detail.ShouldIgnoreTable(t.DDL.Database, t.DDL.Table) {
+		if detail.isIgnoreTxnCommitTs(t) || detail.ShouldIgnoreTable(t.DDL.Database, t.DDL.Table) {
```

Comment:
I think it would be simpler if we require the caller to call `ShouldIgnoreTxn` separately before filtering.
The semantic is not exactly the same:

1. Ignore the whole Txn because it's specified with commit ts
1. OK, the Txn should not be ignored, let's filter something some ignorable mutations

Reply:
refined, PTAL